### PR TITLE
Update AMI ID for Linux slaves

### PR DESCRIPTION
**Pull request by edunham**
Workflow for switching from Daala AWS account to Servo AWS account: 
- [x] Create Buildbot user in Servo account, download its creds, and set up its security groups to match Buildbot user in Daala account
- [x] Copy the buildslave AMI (originally private to the Daala account) to the Servo account (though this changes its ID)
- [x] PR to change all instances of Daala account's copy of the buildslave AMI ID to the Servo account's copy (that's this) 
- [ ] Land PR, update `/srv/pillar/buildbot.sls` `aws-access-key` and `aws-secret-key` to correspond to Servo AWS account's Buildbot user (carefully not running Salt in between these steps)
- [ ] Run Salt on servo-master to deploy changes
- [ ] Fix whatever all this broke
- [ ] Update the wiki

In other words, I'll merge this one by hand at the right time after r+ :)

[Review on Reviewable](https://reviewable.io/reviews/servo/saltfs/183)

### CODE REVIEW

**Comment by larsbergstrom** [comment]
r+

**Comment by edunham** [comment]
Old ID was for an instance in US-West-1. Everything else is in US-West-2. Changed ID to a copy in us-west-2 and will figure out why we're in this region later.

**Comment by larsbergstrom** [comment]
Sounds good to me :-)
